# Fix hard coded DIRAC/utils path in fde_response_mag and fde_response_shield tests
# OCT 7th 2020 by B. Hajgato (UGent)
diff -ru DIRAC-19.0-Source.orig/test/fde_response_mag/test DIRAC-19.0-Source/test/fde_response_mag/test
--- DIRAC-19.0-Source.orig/test/fde_response_mag/test	2019-12-12 17:26:15.000000000 +0100
+++ DIRAC-19.0-Source/test/fde_response_mag/test	2020-10-07 22:41:25.277776298 +0200
@@ -16,7 +16,7 @@
 
 
 test.run(['dc_pbe'],                       ['h2o-1.xyz'],    args='--get="numerical_grid"')
-os.system('../../../utils/fde-mag/dft_to_fde_grid_convert.py')
+os.system('../../../DIRAC*/utils/fde-mag/dft_to_fde_grid_convert.py')
 
 shutil.copy('numerical_grid', 'FILEEX')
 shutil.copy('numerical_grid', 'GRIDOUT')
diff -ru DIRAC-19.0-Source.orig/test/fde_response_shield/test DIRAC-19.0-Source/test/fde_response_shield/test
--- DIRAC-19.0-Source.orig/test/fde_response_shield/test	2019-12-12 17:26:15.000000000 +0100
+++ DIRAC-19.0-Source/test/fde_response_shield/test	2020-10-07 22:41:45.348275614 +0200
@@ -17,7 +17,7 @@
 
 
 test.run(['dc_pbe'],                       ['h2o-1.xyz'],    args='--get="numerical_grid"')
-os.system('../../../utils/fde-mag/dft_to_fde_grid_convert.py')
+os.system('../../../DIRAC*/utils/fde-mag/dft_to_fde_grid_convert.py')
 
 shutil.copy('numerical_grid', 'FILEEX')
 shutil.copy('numerical_grid', 'GRIDOUT')
